### ISSUE TITLE
Is it possible to perform 3D object detection using 2D images with mmdet3d?

### ISSUE THREAD

**Issue by blackCmd** [open-mmlab/mmdetection3d#2522]
Is it possible to perform 3D object detection using 2D images with mmdet3d?

**Comment by sunjiahao1999** [comment]
Of course! Camera only 3D detection is one of the main features of mmdetection3D, you can see the implementation of related methods such as `FCOS3D` and some of the latest fusion methods such as `BEVFusion`

**Comment by parakh08** [comment]
Can you please provide a config file for camera only BEVFusion?

**Comment by snachi2s** [comment]
Is there any implementations in mmdet3d that uses 2D Ground truths(bounding boxes and camera matrix) to estimate 3D bounding box parameters from RGB images?
(Similar to https://paperswithcode.com/paper/3d-bounding-box-estimation-using-deep)
